invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -78,78 +72,68 @@ class DashboardTabs extends Component {
       overrideStore.add(componentId, component);
     }
 
-    this.panes = this.ACTIVE_TAB_PANES.map((pane, index) => {
+    this.menuItems = {};
+    for (const [_, menuItem] of Object.entries(MENU_ITEMS)) {
       const { appId, ...config } = getConfigFromDataAttribute(
         rootElement,
-        _camelCase(pane.configDataAttribute)
+        _camelCase(menuItem.configDataAttribute)
       );
-      return {
-        menuItem: (
-          <Menu.Item key={index} className="selected-menu-tab">
-            {pane.label}
-          </Menu.Item>
-        ),
-        render: () => (
-          <Container>
-            <Tab.Pane>
-              <SearchApp appName={appId} key={appId} config={config} />
-            </Tab.Pane>
-          </Container>
-        ),
+      this.menuItems[menuItem.key] = {
+        menuLabel: menuItem.label,
+        appId: appId,
+        config: config,
       };
-    });
+    }
   }
 
-  onTabChange = (e, data) => {
-    const activePane = this.ACTIVE_TAB_PANES[data.activeIndex];
-    replaceURLPathname(activePane.pathname);
+  handleItemClick = (e, { name }) => {
+    const activeMenuElement = MENU_ITEMS[_lowerCase(name)].key;
+    this.setState({
+      currentActiveMenuElement: activeMenuElement,
+    });
+    replaceURLPathname(_lowerCase(name));
   };
 
-
-
   render() {
-    const Element = Tab;
-    // apply custom menu rendering
-    Element.prototype.renderMenu = renderMenuCustom;
-
-    return (
-      <Container fluid>
-        <Element
-          id="dashboard-tab"
-          defaultActiveIndex={this.state.defaultActiveTab}
-          panes={this.panes}
-          menu={{ secondary: true, pointing: true }}
-          onTabChange={this.onTabChange}
-          renderActiveOnly={true}
-        />
-      </Container>
+    const { currentActiveMenuElement } = this.state;
+    const items = this.menuItems;
+
+    const activeContent = items[currentActiveMenuElement];
+    const menus = Object.keys(items).map(
```

Comment:
See [doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#iterating_through_an_object):
```suggestion
    const menus = Object.keys(items).forEach(([key, value]) => {
        const { menuLabel } = value;
        return (<Menu.Item
            name={menuLabel}
            key={key}
            active={key == currentActiveMenuElement}
            onClick={this.handleItemClick}
          />)}
          );
```